src/main/java/de/ellpeck/rockbottom/api/StaticTileProps.java

```diff
@@ -60,10 +60,19 @@
      */
     public static final IntProp SAPLING_GROWTH = new IntProp("growth", 0, 5);
     /**
-     * Determines how far grown a corn plant is. Used by {@link
-     * GameContent#TILE_CORN}
+     * Determines how far grown a plant is. Used by {@link
+     * GameContent#TILE_CORN} and {@link GameContent#TILE_COTTON}
```

Comment:
If we get further on in the game we shouldn't be specifying everything that uses it. It's fine for now.